### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by yusufokunlola**
Updated pandas and streamlit version

### CODE REVIEW

**Review by Sammybams — APPROVED** [review]
Thank you @yusufokunlola. I will keep in touch. Is 1.3.0 the version you used for scikit-learn?

**Comment by yusufokunlola** [comment]
The Scikit-learn version is 1.1.1
I saw that you added 1.3.0, that's why I did not update it.

**Comment by Sammybams** [comment]
Please kindly update it to what you used. Thanks.

File: experimentation_yusufokunlola/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-streamlit
+streamlit==1.39.0
 scikit-learn==1.3.0
```
Comment (Sammybams) [inline experimentation_yusufokunlola/requirements.txt]:
```suggestion
scikit-learn==1.1.1
```

**Pull request merged**